### PR TITLE
Update prow to include image env for Compliance Operator

### PR DESCRIPTION
**Pull request by Vincent056**
Let's add operator image env to prow, so we use the image built in CI instead of latest for e2e.

### CODE REVIEW

File: ci-operator/config/ComplianceAsCode/compliance-operator/ComplianceAsCode-compliance-operator-master.yaml [hunk 1]
```diff
@@ -128,6 +128,13 @@ tests:
     - as: test-ocp4-pci-dss-node
       cli: latest
       commands: make e2e-rosa
+      dependencies:
+      - env: IMAGE_FROM_CI
+        name: compliance-operator
+      - env: CONTENT_IMAGE_FROM_CI
+        name: testcontent
+      - env: OPENSCAP_IMAGE_FROM_CI
+        name: testopenscap
```
Comment (rhmdnd) [inline ci-operator/config/ComplianceAsCode/compliance-operator/ComplianceAsCode-compliance-operator-master.yaml:137]: The `CONTENT_IMAGE_FROM_CI` and `OPENSCAP_IMAGE_FROM_CI` are not used in any of the rosa tests yet, are they?

Comment (Vincent056) [reply ci-operator/config/ComplianceAsCode/compliance-operator/ComplianceAsCode-compliance-operator-master.yaml:137]: no, I don't think so

**Review by rhmdnd — APPROVED** [review]

**Comment by rhmdnd** [comment]
/pj-rehearse pull-ci-ComplianceAsCode-compliance-operator-master-e2e-rosa

**Comment by rhmdnd** [comment]
The failure is expected due to missing functionality in CO to deploy on ROSA, which we're actively working on.

**Pull request merged**